runtime/parser/init.go

```diff
@@ -46,6 +46,28 @@ mock_users:
 		return err
 	}
 
+	// Determine the driver based on olap
+	var driver string
+	switch olap {
+	case "", "duckdb":
+		driver = "duckdb"
+	case "clickhouse":
+		driver = "clickhouse"
+	default:
+		// Default to duckdb for unknown olap
+		driver = "duckdb"
```

Comment:
This default also happens on line 26 – should probably only have one variable instead of both `driver` and `olap`